packages/devtools/package.json

```diff
@@ -49,5 +49,8 @@
   "devDependencies": {
     "@types/node": "^18.0.0",
     "@types/uuid": "^8.3.0"
+  },
+  "peerDependencies": {
+    "typescript": "^4.6.2"
```

Comment:
Move to the root package.json so that it's aligned for all packages.

Reply:
Also, this would prevent the dependency from being set when installing the package as a user, not as a dev. This is a development dependency only. If we need a specific typescript version then we should install it once and no peer dependencies should be required.

Reply:
I don't think setting `typescript` as peer dep is a good idea, see convo in https://github.com/webdriverio/webdriverio/issues/8699

Reply:
That was exactly the point I was trying to make with my second comment, great to see we are on the same page and agree :)